static/js/jira-server-url.js

```diff
@@ -22,15 +22,11 @@ const checkValidGHEUrl = inputURL => {
 	try {
 		const { protocol, hostname } = new URL(inputURL);
 
-		if (!ALLOWED_PROTOCOLS.includes(protocol)) {
-			setErrorMessage(defaultError);
-			return false;
-		}
-		// This checks whether the hostname whether there is an extension like `.com`, `.net` etc.
-		if (hostname.split('.').length < 2) {
+		if (!/^https?:$/.test(protocol)) {
```

Comment:
Changed this so the validation is the same as the backend and tested with the same inputs from the test. 
<img width="752" alt="Screen Shot 2022-07-04 at 2 06 47 pm" src="https://user-images.githubusercontent.com/37155488/177080158-e9d969f
<img width="715" alt="Screen Shot 2022-07-04 at 2 06 15 pm" src="https://user-images.githubusercontent.com/37155488/177080173-16ece366-ccee-49fd-93a4-03658c0fb391.png">
4-0b8e-46e1-bda1-214c3ed0a465.png">
<img width="811" alt="Screen Shot 2022-07-04 at 2 06 38 pm" src="https://user-images.githubusercontent.com/37155488/177080167-adcf3af1-d30b-448e-a6e3-8d49d1aedc53.png">